crepecake/src/main/java/com/google/cloud/tools/crepecake/json/JsonHelper.java

```diff
@@ -46,7 +46,7 @@
  */
 public class JsonHelper {
```

Comment:
If this class is just dealing with JsonTemplateSerialization, I think we should indicate that in the name. Helper is too generic.